speech/api/SpeechTest/SpeechTest.cs

```diff
@@ -140,10 +141,10 @@ public void TestWithContext()
         [Fact]
         public void TestSyncWords()
         {
-            var output = Run("sync", "-w", _audioRawPath);
+            var output = Run("sync", "-w", s_AUDIO_FILE);
             Assert.Equal(0, output.ExitCode);
-            Assert.Contains("Brooklyn", output.Stdout);
-            Assert.Contains("WordStartTime:", output.Stdout);
+            Assert.Contains("Chromecast", output.Stdout);
+            Assert.Contains("Chromecast", output.Stdout);
```

Comment:
Don't need to assert the same thing twice
```suggestion
            Assert.Contains("Chromecast", output.Stdout);
```